src/fair/stats/__init__.py

```diff
@@ -566,3 +567,70 @@ def sample(self, n: int = 1) -> np.ndarray:
             samples.append(mBeta.sample())
 
         return np.vstack(samples)
+
+
+class GOF(mBeta):
+    """Goodness of fit test for mBeta distribution as described in section 2
+    https://arxiv.org/pdf/2003.06684
+    """
+
+    def __init__(self, null: mBeta, alt: mBeta) -> None:
+        """
+        Args:
+            null (mBeta): null mBeta distribution
+            alt (mBeta): alternative mBeta distribution
+        """
+        self.null = null
+        self.alt = alt
+
+    def _null_ecdf(self, n_samples: int = 100, t_samples: int = 100) -> ECDFResult:
+        """Empirical CDF of null distribution
+
+        Args:
+            n_samples (int, optional): Number of samples to draw. Defaults to 100.
+            t_samples (int, optional): Number of statistics to draw. Defaults to 100.
+
```

Comment:
What do you mean by samples and by statistic in this case?

Reply:
Samples refers to the number of samples from each distribution that we use to compare them. Theoretically you could use a huge number of samples and this would give you the best statistical test, but of course it would be time consuming. So in practice we choose some reasonable size.

Statistics is particular to the approach in the paper. They suggest creating an empirical cdf of the test statistic under the null distribution by sampling the statistic multiple times (that's what's happening in `_null_ecdf`).